### ISSUE TITLE
Make dependencies up-to-date

### ISSUE THREAD

**Issue by galk-in** [clutchski/coffeelint#554]
Hi, @clutchski 
Thx a lot for your module. In project [abao](https://github.com/cybertk/abao) we use [greenkeeper.io](https://greenkeeper.io) for check out-date dependencies. Can you update glob module? Maybe start using greenkeeper.io too.

**Comment by AsaAyers** [comment]
What specifically do you want to get out of updating all of the modules? What specifically is broken that we need to put together a new release with updated modules?

**Comment by galk-in** [comment]
It's not anythings broken, but it has warning. You are using `"glob": "^4.0.0",` In abao is using `"glob": "^7.0.3"`

**Comment by AsaAyers** [comment]
That's fine. Node simply nests our `glob` inside coffeelint. It looks like it's been about 5 months since we had to have a release for bug fixes. Updating dependencies just because they are newer introduces risk of breaking things with no benefits.

Greenkeeper isn't useful because the tests cover CoffeeLint. They rarely, if ever, cover integration with external dependencies because it's difficult to setup those kinds of tests. GreenKeeper just gives a false sense of security.

Finally, CoffeeScript is dead or dying, so I have abandoned the language in favor of ES6 through Babel. I stay subscribed here because I maintained CoffeeLint for more than 2 years and people do occasionally open issues.
